app/src/main/java/com/miraclehwan/detektintegrationexample/FailTest.kt

```diff
@@ -0,0 +1,9 @@
+package com.miraclehwan.detektintegrationexample
+
+data class FailTest (val a:Int,val  b:Boolean,
+
+                     val c:Int){
+    fun testtest( a:Int) {}
```

Comment:
⚠️ **[detekt]** <detekt.EmptyFunctionBlock> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>This empty block of code can be removed.